MusicTaste/MusicTaste.xcworkspace/xcuserdata/amanial.xcuserdatad/xcdebugger/Breakpoints_v2.xcbkptlist

```diff
@@ -0,0 +1,6 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Same for this file; we can gitignore xcuserdata/* so none of the changes in that folder show up in Pull Requests